include/rmm/mr/device/logging_resource_adaptor.hpp

```diff
@@ -158,7 +164,7 @@ class logging_resource_adaptor final : public device_memory_resource {
    * If the upstream allocation is successful logs the
    * following CSV formatted line to the file specified at construction:
    * ```
-   * *TIMESTAMP*,"allocate",*bytes*,*stream*
+   * thread_id,*TIMESTAMP*,"allocate",*bytes*,*stream*
```

Comment:
You added this to the documentation, but where do you actually record the thread ID? This PR doesn't seem to modify do_allocate/do_deallocate where the log is actually written?

Reply:
It's subtle. `spdlog` already natively supports logging `tid`, so all it required was adding the `%t` to the pattern above:
```
 logger_->set_pattern("%t,%H:%M:%S:%f,%v");
```